src/ui/dtl/actions/other/split_data/README.md

```diff
@@ -0,0 +1,61 @@
+# Split Data
+
+`Split Data` splits the input data into datasets based on condition.
+
+### Settings
+
+There are 4 ways of splitting the data:
+
+- **percent**: data will be split by the percentage of total images count.
```

Comment:
Data will be split by the percentage of the total image count. This will result in each dataset containing a proportional number of images based on the specified percentages.